reference/promotions.v3.yml

```diff
@@ -1639,6 +1721,29 @@ components:
       required: true
       schema:
         type: string
+    BeforeCursorQuery:
+      name: before
+      in: query
+      required: false
+      description: >
+        A cursor that can be used for backwards pagination.
+        Will fetch results before the position corresponding to the cursor.
+        Cannot be used with the 'page' query parameter.
+        Cannot be used with the 'after' query parameter.
+      schema:
+        type: string
+
+    AfterCursorQuery:
+      name: after
+      in: query
+      required: false
+      description: >
+        A cursor that can be used for backwards pagination.
```

Comment:
probably typo ?
```suggestion
        A cursor that can be used for forwards pagination.
```

Reply:
Ooops, thanks for this catch @donald-nguyen-bc  🙇